README.md

```diff
@@ -72,143 +103,12 @@ Alternatively, you can use [catppuccin-gtk](https://github.com/NixOS/nixpkgs/blo
-
-### GDM Theme
-
-> [!WARNING]
-> Applying a custom theme to GDM is not recommended as it is not themeable, however you can do it through certain *hacks*.
-
-To apply the theme to GDM, A new `gnome-shell-theme.gresource.xml` needs to be complied.
-To achieve this, you can run the following:
-
-```bash
-# Backup the current gresource file.
-sudo cp -av /usr/share/gnome-shell/gnome-shell-theme.gresource{,~}
-sudo glib-compile-resources --target="/usr/share/gnome-shell/gnome-shell-theme.gresource" --sourcedir="$THEME_DIR" "$THEME_DIR/gnome-shell-theme.gresource.xml"
-```
-
-Make sure to replace `$THEME_DIR` to where the theme was extracted accordingly.
-
-- For nix users, it'll be the nix store path of the package.
-- For AUR users, it'll be in `~/.themes`
-- Otherwise, it'll be wherever you extracted the theme.
-
-### Using the script
-
-**Note**: Ensure that you have at least Python version 3.10 installed
-
-Set up the installer using
-
-```bash
-git clone --recurse-submodules git@github.com:catppuccin/gtk.git
-cd gtk
-virtualenv -p python3 venv  # to be created only once and only if you need a virtual env
-source venv/bin/activate
-pip install -r requirements.txt
-```
-
-To check out the install script, run
-
-```bash
-python install.py --help
-```
-
-> Tip: `python install.py --help` allows the following options:
-
-```
-Compulsory field        Specify color variant(s) [mocha|frappe|macchiato|latte|all]
--d, --dest DIR          Specify destination directory (Default: ~/.themes)
--n, --name NAME         Specify theme name (Default: Colloid)
--a, --accent VARIANT... Specify theme color variant(s) [rosewater|flamingo|pink|mauve|red|maroon|peach|yellow|green|teal|sky|
-                        sapphire|blue|lavender|all] (Default: blue)
--s, --size VARIANT...   Specify size variant [standard|compact] (Default: standard variant)
--l, --link              Link installed gtk-4.0 theme to config folder for all libadwaita app use this theme
---zip                   Zips up the finally produced themes.
---tweaks                Specify versions for tweaks [black|rimless|normal|float]
-                        1. black:    Blackness color version
-                        2. rimless:  Remove the 1px border about windows and menus
-                        3. normal:   Normal windows button style (titlebuttons: max/min/close)
-                        4. float:    Floating gnome-shell panel style
--h, --help              Show help
-```
-
-You can install any theme like the following example
-
-```bash
-python install.py mocha -a sky --tweaks rimless -d ~/.themes
-
-```
-
-You can build all possible variations of the theme possible using the following command and it will install it to releases folder
-
-```bash
-python install.py all -a all
-```
-
-## Development
-
-You need to install the following packages to build the theme. Check with your distribution for the package names in the repository
-
-- `sassc`
-- `inkscape`
-- `optipng`
+### For Other Distros
```

Comment:
Wouldn't this not be needed since the manual instructions are available above and that is what this section links to?

Reply:
i think so yeah

Reply:
fixed